x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -1233,7 +1233,12 @@ public enum Cap {
         /**
          * Support avg with aggregate metric doubles
          */
-        AGGREGATE_METRIC_DOUBLE_AVG(AGGREGATE_METRIC_DOUBLE_FEATURE_FLAG);
+        AGGREGATE_METRIC_DOUBLE_AVG(AGGREGATE_METRIC_DOUBLE_FEATURE_FLAG),
```

Comment:
Why add this to the end? It would be better to add this to after the other AGGREGATE_METRIC_DOUBLE capabilities, around line 892. That would also reduce the chance of conflicts with others also mistakenly adding to the end of the enum.